libr/core/panels.c

```diff
@@ -2946,7 +2946,7 @@ static void savePanelsLayout(RCore* core, bool temp) {
 		pj_kn (pj, "y", panel->pos.y);
 		pj_kn (pj, "w", panel->pos.w);
 		pj_kn (pj, "h", panel->pos.h);
-		pj_kb (pj, "caching", panel->caching);
+		pj_kn (pj, "caching", panel->caching);
```

Comment:
n expects ut64, this is int, and i dont see why you want caching to be int instead of bool

Reply:
so how do you pass char* "true" as bool true, i mean i can do it somehow but i can save it if i make it int at the first place